locksmith/src/websub/jobs/expiredKeys.ts

```diff
@@ -0,0 +1,104 @@
+import networks from '@unlock-protocol/networks'
+import config from '../../config/config'
+import { SubgraphService } from '@unlock-protocol/unlock-js'
+import { logger } from '../../logger'
+import { hasReminderAlreadySent } from '../../operations/keyExpirationReminderOperations'
+import { sendEmail } from '../../operations/wedlocksOperations'
+import * as userMetadataOperations from './../../operations/userMetadataOperations'
+import * as Normalizer from '../../utils/normalizer'
+
+function notifyExpiredKeys({
+  keys,
+  network,
+}: {
+  keys: any[]
+  network: number
+}) {
+  return keys?.map(async (key: any) => {
```

Comment:
don't use map, but `for` because it is async and you don't want all promises to execute at once